### PR TITLE
feat: add manage tax configurations component

### PR DESCRIPTION
**Pull request by jonathanzhang53**
## Description
Implemented Manage Tax Configurations List View described in issue #329 

## Screenshots, if any
tax

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/products/manage-tax-configurations/manage-tax-configurations.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+
+/** Custom Services */
```
Comment (abhaychawla) [inline src/app/products/manage-tax-configurations/manage-tax-configurations.component.ts]: Unnecessary import.

File: src/app/products/products-routing.module.ts [hunk 2]
```diff
@@ -1,43 +1,49 @@
-/** Angular Imports */
```
Comment (abhaychawla) [inline src/app/products/products-routing.module.ts]: Same lines have been removed and added here again, please ensure only the required new additions are present in the PR.

File: src/app/products/products.component.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 /** Angular Imports */
 import { Component, OnInit } from '@angular/core';
 
+/** Custom Services */
```
Comment (abhaychawla) [inline src/app/products/products.component.ts]: Unnecessary changes not required for this PR. Please remove them.

File: src/app/products/products-routing.module.ts (hunk 2 above)
Comment (jonathanzhang53) [reply src/app/products/products-routing.module.ts]: @abhaychawla I'm not sure what the issue is here on my end. It shows everything as changed and I can't get it to just show specific changes.

Comment (jonathanzhang53) [reply src/app/products/products-routing.module.ts]: Let me try something

**Comment by abhaychawla** [comment]
The import for products service still exists. It needs to be removed for now.

**Comment by jonathanzhang53** [comment]
@abhaychawla fixed

**Comment by abhaychawla** [comment]
Your commit message has changed.

**Comment by jonathanzhang53** [comment]
Sorry I thought list view was a better name. Changing it back.

**Comment by abhaychawla** [comment]
You may submit it now, we are trying to have consistency in commit messages so that why.

**Comment by jonathanzhang53** [comment]
@abhaychawla Don't worry. I welcome the change requests because I'm definitely new to open source. It'll be nice for me in future PRs to not make the same mistakes :)
